.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
-script: npm run coverage && cat ./coverage/lcov.info | coveralls
+after_script: npm run coverage && cat ./coverage/lcov.info | coveralls
```

Comment:
this seems slightly wrong --- I think `npm run coverage` does the actual testing stuff. Probably just want to move the `cat ./coverage/lcov.info | coveralls` to after_script